pkg/metrics/otel.go

```diff
@@ -151,14 +157,14 @@ service:
   pipelines:
     metrics/cloudmonitoring:
       receivers: [opencensus]
-      processors: [batch, filter/cloudmonitoring]
+      processors: [filter/cloudmonitoring, resourcedetection, batch]
```

Comment:
Is it necessary for `batch` be the last processor? If yes, can you add a comment?

Reply:
Just retried and no it is not, moving it back.